src/platform/backends/shared/base_virtual_machine.h

```diff
@@ -84,6 +84,10 @@ class BaseVirtualMachine : public VirtualMachine
     void load_snapshots() override;
     std::vector<std::string> get_childrens_names(const Snapshot* parent) const override;
     int get_snapshot_count() const override;
+    void remove_snapshots_from_image() const override
+    {
+        throw NotImplementedOnThisBackendException("remove all snapshots from the image");
```

Comment:
Would this be considered an extension of the "snapshots" feature and therefore be labelled as 
```
throw NotImplementedOnThisBackendException("snapshots");
```

Reply:
This can be done since the backends of remove_snapshots_from_image are indeed in line with the snapshot  backends.